.vscode/extensions.json

```diff
@@ -1,3 +1,16 @@
 {
-  "recommendations": ["biomejs.biome"]
+  "recommendations": [
+    "biomejs.biome",
+    "naumovs.color-highlight",
+    "usernamehw.errorlens",
+    "eamodio.gitlens",
+    "VisualStudioExptTeam.vscodeintellicode",
+    "esbenp.prettier-vscode",
+    "Prisma.prisma",
+    "bradlc.vscode-tailwindcss",
+    "vincaslt.highlight-matching-tag",
+    "YoavBls.pretty-ts-errors",
+    "dbaeumer.vscode-eslint",
+    "streetsidesoftware.code-spell-checker"
```

Comment:
Any reason for all of these?

Reply:
They're recommendations for a better developer experience. Code Spell Checker helps you to write correct English and points out spelling mistakes